aicsimageio/metadata/README.md

```diff
@@ -1,13 +1,11 @@
 # AICSImage Metadata Module
 
-Undecided how to organize this module. This likely isn't included in 4.0 but more
-likely 4.1 or similar but good to start thinking about.
+Planned layout of the `metadata` submodule
 
-I am currently thinking:
 ```
 aicsimageio/
     metadata/
-        omexml.py  # our remade and updated omexml object generated from source XSD
+        utils.py  # common utilities and cleaning
         transform.py  # the transform functions to convert to ome
```

Comment:
i would suggest separate subdirectories for the proprietary format conversions instead but I know that is down the road a bit, still.

Reply:
Yea we can decide that when we get the first translator in imo.